pyomo/contrib/doe/doe.py

```diff
@@ -822,7 +822,7 @@ def run_grid_search(
 
                 # give run information at each iteration
                 self.logger.info(
-                    'This is the  %s run out of  %s run.', (count + 1), total_count
+                    'This is %s run out of %s.', (count + 1), total_count
```

Comment:
```suggestion
                    'This is %s run out of %s.', count, total_count
```
Looks like you fixed the grammar but not the off-by-one issue